src/ProjectTemplates/Web.ProjectTemplates/content/Worker-CSharp/.template.config/cs-CZ/strings.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": "1.0.0.0",
   "strings": {
-    "name": "Služba pracovního procesu",
-    "description": "Šablona prázdného projektu pro vytvoření služby pracovního procesu"
+    "name": "Worker Service",
```

Comment:
It seems that the name `Worker Service` is not getting translated anymore. Expected?

Reply:
If it is branding, or a term in term studio, yes. More likely, it could be due to rolling translations coming back since this is a fresh onboarding of these files.